pkg/meta/tkv.go

```diff
@@ -3380,3 +3387,37 @@ func (m *kvMeta) doAttachDirNode(ctx Context, parent Ino, inode Ino, name string
 		return nil
 	}, parent))
 }
+
+func (m *kvMeta) touchAtime(_ctx Context, ino Ino, cur *Attr) syscall.Errno {
+	if (m.conf.AtimeMode != StrictAtime && m.conf.AtimeMode != RelAtime) || m.conf.ReadOnly {
+		return 0
+	}
+
+	attr := &Attr{}
```

Comment:
adjust it like that in RedisMeta

Reply:
I agree, but I don't know how to update a key in tkv without transaction, seems only `tikvTxn` implements `set` method, which calls `KVTxn.Set()`. I didn't find other set implementations outside transaction.

Reply:
> 

Ok, let me try.

Reply:
Fixed